### PR TITLE
ENH added webservices action

### PR DESCRIPTION
**Pull request by beckermr**
This PR adds the config file for the webservices github action.

Checklist
* [x] Added a ``news`` entry

### CODE REVIEW

**Comment by beckermr** [comment]
@isuruf Care to review and merge?

**Comment by beckermr** [comment]
cc @conda-forge/core

**Comment by isuruf** [comment]
You need to add this file to https://github.com/conda-forge/conda-smithy/blob/master/conda_smithy/templates/conda-forge.yml.ci-skel.tmpl#L11
Or extend the skipping to directories.

**Comment by beckermr** [comment]
Done, but I am very confused by this option. 

Why do we want to skip rerendering the files in that list? The README and build*.py files should change when we rerender.

File: conda_smithy/templates/conda-forge.yml.ci-skel.tmpl [hunk 1]
```diff
@@ -9,3 +9,4 @@ skip_render:
   - .github/CONTRIBUTING.md
   - .github/ISSUE_TEMPLATE.md
   - .github/PULL_REQUEST_TEMPLATE.md
+  - .github/workflows/webservices.yml
```
Comment (isuruf) [inline conda_smithy/templates/conda-forge.yml.ci-skel.tmpl]: Does the following work? I think it should.
```suggestion
  - .github/workflows
```

Comment (beckermr) [reply conda_smithy/templates/conda-forge.yml.ci-skel.tmpl]: But if we change the automerge options we want to remove that file so we cannot skip it on rerendering, right?

**Comment by isuruf** [comment]
See https://conda-forge.org/docs/user/ci-skeleton.html

**Comment by beckermr** [comment]
Ditto for changes to the action itself.

**Comment by beckermr** [comment]
ok @isuruf I think this is ready to go!

**Comment by isuruf** [comment]
Can you add `.github/workflows` to `test_files_skip_render` just to see if that works?

**Comment by beckermr** [comment]
Alright, check out the latest commit. The trailing slash was causing issues. This appears to work now.

**Comment by isuruf** [comment]
Thanks

**Pull request merged**